src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,39 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { getUser } from '../../api';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  isLoading: boolean,
+  setIsLoadingUser: (value: boolean) => void
+  onClose: (value: number) => void
+  currentTodo: Todo,
+};
+
+export const TodoModal: React.FC<Props> = ({
+  isLoading,
+  setIsLoadingUser,
+  onClose,
+  currentTodo,
+}) => {
+  const userInitial = {
+    name: '',
+    email: '',
+  };
+  const [user, setUser] = useState(userInitial);
+
+  useEffect(() => {
+    getUser(currentTodo.userId).then((currentUser) => {
+      setUser(currentUser);
+      setIsLoadingUser(false);
+    });
```

Comment:
what if getUser will return an error? You should handle errors here
Also async await are better)